Cargo.toml

```diff
@@ -44,9 +44,9 @@ postgres-types = { version = "0.2", features = [
 chrono = { version = "0.4", features = ["std"], optional = true }
 
 [features]
-default = ["server-api-ring"]
-server-api-ring = ["server-api", "ring"]
-server-api-aws-lc-rs = ["server-api", "aws-lc-rs"]
+default = ["server-api"]
```

Comment:
scram should probably be enabled by default

Reply:
Thanks for the notes. I will be thinking about that.